packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -501,10 +503,15 @@ const schema: SchemaType<"Tags"> = {
     canCreate: ['admins', 'sunshineRegiment'],
     optional: true,
     group: formGroups.advancedOptions,
+    control: 'checkboxgroup',
+    label: "Who can add this tag to things or vote on its relevance?",
```

Comment:
Actually I just noticed that there's a related issue. If you select an option, save the tag, then unselect it and save the tag again, `canVoteOnRels` gets set to `{}` and then no one can vote on the tag. I guess since we were just manually updating this field we avoided this issue.

Sorry this is kind of an annoying issue that you prob don't want to deal with... Maybe you can set the field to `null` instead? If not then you'll have to look at `tagRelVoteRules.tsx` and fix all the relevant logic (in particular `canVoteOnTag()`).

Reply:
tooltip seems good to have regardless.

I wonder if there is a simple way to make "unselect everything sets this to null vs empty list work 🤔

Reply:
I guess I can have an `onUpdate` that does that, but I don't really like it.

Another option here is to give this field a default value (all groups/or groups you listed above?)

Reply:
having the default value results in a most straightforward/intuitive ui behavior. but it does require migration and maybe future migrations

other options are (am I missing anything?):
- leave it be as it is rn
- track usages and update them to handle empty array
- have onUpdate trigger that sets this to null if we get empty array (leaning towards this rn if we're not doing the default value)

Reply:
@Stvad I think the simplest way forward here is to gate the input field (so that it only appears on Waking Up). Then you can decide which of the solutions you prefer to have there. (Though if you go the migration route that would still affect all forums so I would prefer you not pick that one)